sdk/storage/azfile/internal/generated/autorest.md

```diff
@@ -123,3 +123,49 @@ directive:
     $.FileContentType["x-ms-parameter-grouping"].name = "share-file-http-headers";
```

Comment:
For acronyms on the generated layer you'll need to use transforms, please refer to blobs package

Reply:
This is also an autorest transform. Can you share what exactly you are referring to?